act-rules-format.md

```diff
@@ -254,6 +256,28 @@ If the rule does not map to any [=accessibility requirement=], the accessibility
 If the `failed` [=outcome=] cannot be mapped to an [=accessibility requirement=], there <em class="rfc2119">must not</em> be an accessibility requirement in the accessibility requirements mapping. The optional [Background section](#background) could be used to list [=accessibility requirements documents=] when they are thematically related, but for which the rule is not a failure test.
 
 
+### Implementing Accessibility Requirement Mappings ### {#implementing-accessibility-requirement-mappings}
+
+[=Implementations=] can only have a [=complete=] [=consistency=] level if they reports accessibility requirements as *not satisfied* in a way that is consistent with the accessibility requirements mapping. A set of [=implementation procedures=] has a <dfn>consistent accessibility requirements mapping</dfn> when all `failed` test cases of an ACT Rule are reported as *not satisfied* for each [=relevant accessibility requirement=] by at least one implementation procedure in the set. Additionally, no other accessibility requirement in the same [=accessibility requirements documents=] is reported as *not satisfied* by the implementation procedures in the set.
+
+A <dfn>relevant accessibility requirement</dfn> is an accessibility requirement in the [accessibility requirements mapping](#accessibility-requirements-mapping) of an ACT Rule that is part of an [=accessibility requirements document=] and conformance level that an implementation was designed to test for.
+
+<aside class=example>
+  <header>Example of relevant accessibility requirements:</header>
+  <blockquote>
+    <p>Example of relevant accessibility requirements: An ACT Rule includes the following fictional WCAG 2 success criteria:</p>
+    <ul>
+      <li>WCAG 2.9, Success criterion 5.1.1 Cool content (minimal), level A</li>
+      <li>WCAG 2.9, Success criterion 5.1.2 Cool content (enhanced), level AA</li>
+      <li>WCAG 2.9, Success criterion 5.1.3 Cool content (no exceptions), level AAA</li>
+      <li>WCAG 2.10, Success criterion 5.2.1 Cooler content, level A</li>
+    </ul>
+    <p>The relevant accessibility requirements for an implementation that tests WCAG 2.9 level A and level AA would be 5.1.1 and 5.1.2. Criterion 5.1.3 is not relevant because it is at a higher conformance level. Criterion 5.2.1 is not relevant because it is part of WCAG 2.10, which the implementation does not report on.</p>
+    <p>To have a completely consistent implementation therefore, this implementation has to report 5.1.1 and 5.1.2 as not satisfied on all failed test cases.</p>
```

Comment:
```suggestion
    <p>To have a completely consistent implementation therefore, this implementation has to report 5.1.1 and 5.1.2 as *not satisfied* on all failed test cases.</p>
```
to keep consistency with the paragraph at https://github.com/w3c/wcag-act/pull/522/files#diff-9683e9f77e5bf32de8ddf25c99bd53060db70b88fc7d1a708f68346cdd0fded7R261